Sources/EdgeBridge.swift

```diff
@@ -36,6 +38,14 @@ public class EdgeBridge: NSObject, Extension {
         registerListener(type: EventType.rulesEngine,
                          source: EventSource.responseContent,
                          listener: handleRuleEngineResponse)
+
+        registerListener(type: EventType.edgeBridge,
+                         source: EventSource.startCapture,
+                         listener: handleCaptureRequest)
```

Comment:
I think we can have source as `EventSource.captureContent`. And in the data we can have a key `state:start` which means start capturing and `state:stop` which means stop capturing. This way we can make the listener generic and consolidated into one.